src/csvcubed/models/validationerror.py

```diff
@@ -60,3 +60,11 @@ class PydanticThrowableSpecificValidationError(
     """
 
     pass
+
+@dataclass
+class ValidateModelProperiesError(ValidationError):
+
+    """
+    This error will be returned with the name of the class attribute.
```

Comment:
Can you explain a bit better the situation/context in which this error will be generated?

Reply:
done